### ISSUE TITLE
Find files in same package but different source root

### ISSUE THREAD

**Issue by markiewb** [markiewb/nb-resource-hyperlink-at-cursor#10]
Given
```
@ContextConfiguration("MyTest-context.xml")
``` 
in 
```
src/test/java/com/foo/MyTest.java
```
also match
```
src/test/resources/com/foo/MyTest-context.xml
```

**Comment by markiewb** [comment]
fixed in 1.3.0.2-SN

**Comment by markiewb** [comment]
Fixed NPE in 1.3.0.4-SN  
d9c2822256b50a80a464fffbfd75fd39d3ca58e5
